util/opentelemetry-util-genai/tests/test_upload.py

```diff
@@ -162,6 +162,21 @@ def test_upload_then_shutdown(self):
             "should have uploaded 3 files",
         )
 
+    def test_upload_when_inputs_outputs_empty(self):
+        self.hook.on_completion(
+            inputs=[],
+            outputs=[],
+            system_instruction=FAKE_SYSTEM_INSTRUCTION,
+        )
+        # all items should be consumed
+        self.hook.shutdown()
+
+        self.assertEqual(
+            self.mock_fs.open.call_count,
+            1,
+            "should have uploaded 1 file",
+        )
```

Comment:
can you add an assertion that the empty ones don't create a ref on the span or log?